src/layout/Tabbed.js

```diff
@@ -69,6 +69,11 @@
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
 
+        if (this._tabContainer) {this._tabContainer.remove();}
+
+        this._tabContainer = element.insert("div", this.tabLocation() === "bottom" ? null : "div");
+        this._tabContainer.classed("on_" + this.tabLocation(), true);
```

Comment:
This doesn't work as you will end up with several classes if the user changes tab location at runtime, you need:
```
this._tabContainer.attr("class", "on_" + this.tabLocation());
```